text-run/contains-make-task.js

```diff
@@ -0,0 +1,26 @@
+const makeTools = require("./tools/make-tools");
```

Comment:
Is it possible to consume these from a remote location or are these specific to this repository?